performant-software/DM
# resource list modal opens on project load, resource list reorderable …

…(non-persisting), annotation hover popup resizable (non-persisting)

### What does this PR do?
- Triggers resource list modal show upon loading a project from the URL-included shortcut
- Makes resources in list reorderable through drag and drop; includes helper methods for persisting order, but these are disabled pending full integration with backend
- Makes the annotation list popup resizable; does not yet persist the new size

### What issues does this address?
Closes #28, addresses #33, addresses #24 

### How to test?
- Go to http://67.207.158.85/workspace/#6kgzvltt and confirm that the resource list modal appears after initial page load
- Log in and reopen the resources list by clicking the project name (or choosing a new one). Try to reorder the resource rows by dragging them by their 3-bar reorder icons (see attached image)
- Add an image resource with annotations, such as the first that shows up in the resource list at http://67.207.158.85/workspace/#6kgzvltt. Hover over an annotation highlight in the image so that the annotation list popup appears - try resizing the popup by dragging the lower-right corner.
<img width="607" alt="screen shot 2017-04-06 at 2 30 05 pm" src="https://cloud.githubusercontent.com/assets/2431978/24772075/aa36ed12-1ad5-11e7-9aad-a7f761049327.png">
